_pages/community.html

```diff
@@ -10,7 +10,7 @@
   <div class="container">
     <div class="heading__body">
       <div class="heading__headline common-headline">
-        <h1>We make Rails better.</h1>
+        <h1>The community makes Rails better.</h1>
```

Comment:
```suggestion
        <h1>We make Rails better.</h1>
```

Let's keep the original for messaging purposes. I know the page is called community, and we do want it to include everyone who contributes, but the "we" sounds better here.

Reply:
Sure thing!